src/cli/tools/protoc.ts

```diff
@@ -0,0 +1,69 @@
+import path from 'node:path';
+import { execa } from 'execa';
+import { inject, injectable } from 'inversify';
+import { BaseInstallService } from '../install-tool/base-install.service';
+import {
+  CompressionService,
+  EnvService,
+  HttpService,
+  PathService,
+} from '../services';
+
+@injectable()
+export class ProtocInstallService extends BaseInstallService {
+  readonly name = 'protoc';
+
+  private get ghArch(): string {
+    switch (this.envSvc.arch) {
+      case 'arm64':
+        return 'aarch_64';
+      case 'amd64':
+        return 'x86_64';
+    }
+  }
+
+  private version(version: string): string {
+    // Remove patch version from version string
+    // since the download URL does not include it
+    return version.split('.').slice(0, 2).join('.');
```

Comment:
remove this function, pass the short versions as is.

you probably need to override the validate function as the short version isn't semver

Reply:
are you sure? didn't want to break the validation process

Reply:
fixed